loren/load.py

```diff
@@ -4,6 +4,7 @@
 # pylint: disable=missing-function-docstring
 try:
     import json
+    import yaml
```

Comment:
yaml is used in so many places it doesn't need to be in the try/except ... jsonschema on the other hand is not something you need installed to run loren, but it is nice to